src/coreclr/jit/llvm.cpp

```diff
@@ -317,6 +317,7 @@ Type* getLLVMTypeForVarType(var_types type)
     switch (type)
     {
         case var_types::TYP_BOOL:
+            return Type::getInt1Ty(_llvmContext);
```

Comment:
Note that `TYP_BOOL` is a bit of a lie. There is no actual requirement for `TYP_BOOL` variables to have only the lowest bit set (there is some frontend code which assumes this, it will gradually be made not to do that).

It is safest to assume `TYP_BOOL` is the same as `TYP_UBYTE`.

Reply:
Thanks have mapped `TYP_BOOL` as suggested.  I thought this would be more involved due to LLVM wanting i1s for its branch instructions but the tests are passing so guess not.